src/contracts/definitions/snip-721-def.ts

```diff
@@ -12,17 +12,22 @@ export const snip721Def: ContractDefinition = {
       return { contract_info: {} };
     },
 
-    getNumTokens({ address, key: viewing_key }: Context): ContractQueryRequest {
-      const viewer = { address, viewing_key };
+    getNumTokens(
+      { address, key: viewing_key }: Context,
+      sendViewer?: boolean
+    ): ContractQueryRequest {
+      let viewer = sendViewer ? { address, viewing_key } : null;
```

Comment:
`let` is used for mutable variables... why don't use `const` for immutable?

Reply:
no reason, we can switch to `const`, working on it

Reply:
Yeah, all queries and messages are complete and there is no optional VK so we need to pass always a VK